tools/pylib/_boutcore_build/boutcpp.pxd.in

```diff
@@ -32,6 +32,31 @@ cat <<EOF
     Field3D pow(Field3D,double)
     Field3D & ddt(Field3D)
 
+cdef extern from "field2d.hxx":
```

Comment:
I guess this is basically the same as for field3d?
Could this be done by using a for loop:
```
for field in Field2D Field3D
do ... ; done
```
This will make it easier to add
* more fields
* functions